app/main.py

```diff
@@ -1 +1,39 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
```

Comment:
If you are separating parameters into lines - it's better to have one parameter for one line.